docs/signer.md

```diff
@@ -0,0 +1,354 @@
+# Kritis Signer
+
+Kritis Signer is a command-line tool that creates attestations. These attestations can be used for Kritis and Binary Authorization deployment enforcement.
+
+The tool can create attestations directly or conditionally, checking policy conformance based on Vulnerability scanning results from Google Cloud Container Analysis.
+
+It can be run either locally or as part of a continuous integration (CI) pipeline.
+
+Currently, the tool supports vulnerability-based policy check, 
+using [Google Vulnerability Scanning](https://cloud.google.com/container-registry/docs/vulnerability-scanning), a feature of  [Google Container Analysis](https://cloud.google.com/container-registry/docs/container-analysis) that returns a list of vulnerabilities associated with a container image. Kritis Signer then creates an attestation that is stored in the Google Container Analysis data store.
+Support for other type of checks (e.g., base-image check), other vulnerability sources and attestation storage are underway.
```

Comment:
In Cloud docs, we avoid future reference.

Reply:
It should be fine to have future references here, and it also gives better idea of the tool. WDYT?

Reply:
Rereading, I'm not sure what underway is.  If this means that: "Container Analysis regularly updates their database of published vulnerabilities sources", that might be good to communicate, along with a link to the Vulnerability sources section of the CA docs.  If, by "attestation storage" you are communicating that, "these vulnerabilities are stored in Container Analysis as Notes."  That could be good to add, but then be clear about when Note refers to  vulnerability descriptor (?) for Container Analysis and when it refers to the attestor Note name to which the user is attaching an attestation for BinAuthz.

Reply:
One more way to think about future references is: how does it help the user to know that something will happen at some as yet undetermined point in the future.

Reply:
Many other customers use e.g., Twistlock for vulnerability scanning, and store attestations in e.g., Grafeas or other storage systems. Alternative vulnerability sources and storage are supported in other signer solutions like Voucher. This is to say our Signer solution is a general open source solution, as opposed to a specific GCP solution.

Looking at another way is that we do plan to add more support. Stating it here earlier would avoid swaying away potential users.

Reply:
Then maybe state that either here or at the top of the doc. This signer is provided as open source so it can be adapted to be integrated with any product.